code/game/objects/items/devices/geiger_counter.dm

```diff
@@ -147,7 +147,7 @@
 			var/total = 0
 			for(var/i in get_rad_contents(target))
 				var/atom/scanning = i
-				var/datum/component/radioactive/R = GET_COMPONENT_FROM(/datum/component/radioactive, scanning)
+				var/datum/component/radioactive/R = scanning.GetComponent(/datum/component/radioactive)
```

Comment:
GET_COMPONENT_FROM(R,/datum/component/radioactive, scanning)

Whole line, not just the right side.

Reply:
i know but it's the same thing

Reply:
what's the benefit @Barhandar ?

Reply:
ask Cyberboss